.astropybot.yml

```diff
@@ -0,0 +1,2 @@
+CHANGELOG_CHECK: false
+AUTOCLOSE_STALE_PULL_REQUEST: false
```

Comment:
I would be in favor of not capitalizing options in the config yml file (consistent with other yml files we use for various CI services)